solutions/CSES Planets Queries I.cpp

```diff
@@ -0,0 +1,59 @@
+---
+id: cses-1750
+source: CSES
+title: Planet Queries I
+author: Dong Liu
+---
+
+Simply construct a parent matrix (in powers of 2), then answer each query by jumping from the starting node based on the binary representation of the distance.
```

Comment:
I think you should mention binary jumping somewhere here, since "parent matrix" sounds like you're just storing their immediate parent and you don't say where you jump to for each query